cli/src/command/wallet.rs

```diff
@@ -196,6 +198,15 @@ pub async fn new_account_command(
     Ok((wallet, alias))
 }
 
+pub async fn node_info_command(storage_path: &Path, snapshot_path: &Path) -> Result<Wallet, Error> {
+    let password = get_password("Stronghold password", !snapshot_path.exists())?;
```

Comment:
We don't really need the secret manager for this so it's a bit annoying to request a password. I would modify `unlock_wallet` to take an `Option<Password>` and conditionally create a secret manager. And pass None for this command.

Reply:
yeah, totally agree that is annoying. I didn't know that is possible, fixed.